hack/bin/integration-setup-federation.sh

```diff
@@ -49,7 +49,21 @@ export FEDERATION_DOMAIN_2="federation-test-helper.$FEDERATION_DOMAIN_BASE"
 
 echo "Installing charts..."
 
+set +e
 helmfile --environment "$HELMFILE_ENV" --file "${TOP_LEVEL}/hack/helmfile.yaml" sync --skip-deps --concurrency 0
+EXIT_CODE=$?
+
+echo "!! Helm install failed. Attempting to get some more information ..."
+
+kubectl -n "$NAMESPACE_1" get events | grep -v "Normal "
+kubectl -n "$NAMESPACE_2" get events | grep -v "Normal "
+${DIR}/kubectl-get-debug-info.sh "$NAMESPACE_1"
+${DIR}./kubectl-get-debug-info.sh "$NAMESPACE_2"
```

Comment:
shouldn't all this go into the "if exit_code > 0 then..."?

Reply:
yes, correct, sorry 🙈